sp/src/game/server/hl2/npc_antlion.cpp

```diff
@@ -3206,8 +3206,13 @@ bool CNPC_Antlion::HandleInteraction( int interactionType, void *data, CBaseComb
 		Flip();
 
 		// If the Combine soldier is a player, don't deal damage on top of flipping
+		// No, at the beginning of Ashes the non-damaging melee would be a disappointment like in Doom Eternal - HEVcrab
+
 		if ( sender->IsPlayer() )
 		{
+			float dmg = sk_antlion_health.GetFloat()/5; // DO deal damage of 1/5 Antlion's max health to kill in 5 kicks
```

Comment:
I would like all changes to be toggleable so that we can configure them in skill.cfg or in another way. I have two requests for this.

Replace "5" with a ConVar for number of kicks to kill. Maybe call it sk_antlion_kicks_to_kill or something like that.

If sk_antlion_kicks_to_kill is 0 (or -1 if you prefer), skip this check and do no damage. Default the ConVar to this value. We will set it to 5 in skill.cfg in an update.

Reply:
OK, I will